components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -500,6 +480,25 @@ const InsightPage = ({ edit, insight, pageRepos }: InsightPageProps) => {
             );
           })}
         </RepositoriesCart>
+        <Title className="!text-1xl !leading-none mb-4 mt-8" level={4}>
```

Comment:
IMHO we must make a wrapper upon title and `<div className="flex justify-between"></..>` and put this. `mt-8 pt-6 border-t` or we can added here these classes. To see the dividing line.

Reply:
Ok, on it.